tests/unit/type/set.tcl

```diff
@@ -1029,6 +1029,13 @@ foreach type {single multiple single_multiple} {
         r srem $myset {*}$members
     }
 
+    proc completed_rehashing_ht_size_N_mem_M {myset n m} {
+        set htstats [r debug HTSTATS-KEY $myset]
+        assert {![string match {*rehashing target*} $htstats]}
+        return {[string match {*table size: $n*} $htstats] 
+                && [string match {*number of elements: $m*} $htstats]}
+    }
```

Comment:
method name and variable name update suggestion:

```suggestion
    proc verify_rehashing_completed_key {myset table_size keys} {
        set htstats [r debug HTSTATS-KEY $myset]
        assert {![string match {*rehashing target*} $htstats]}
        return {[string match {*table size: $table_size*number of elements: $keys*} $htstats]
    }
```

Also check if there are other places with similar check, you could update them.